i18n/de/LC_MESSAGES/ckanext-switzerland.po

```diff
@@ -617,11 +609,84 @@ msgid "Organization"
 msgstr "Organisation"
 
 #: ckanext/switzerland/templates/snippets/stream.html:11
-#: ckanext/switzerland/templates/snippets/stream.html:18
-#: ckanext/switzerland/templates/snippets/stream.html:31
+#: ckanext/switzerland/templates/snippets/stream.html:19
+#: ckanext/switzerland/templates/snippets/stream.html:32
 msgid "unknown"
 msgstr "unbekannt"
 
+#: ckanext/switzerland/templates/snippets/stream.html:62
+msgid "No activity found for this type"
+msgstr "Keine Aktivität für diesen Typ gefunden"
+
+#: ckanext/switzerland/templates/snippets/changes/resource_extras.html:5
+msgid ""
+"Added field <q>{key}</q> with value <q>{value}</q> to resource "
+"{resource_link} in {pkg_link}"
+msgstr ""
+"Feld <q>{key}</q> hinzugefügt mit dem Wert  <q>{value}</q> zur Ressource "
+"{resource_link} in {pkg_link}"
+
+#: ckanext/switzerland/templates/snippets/changes/resource_extras.html:16
+msgid "Added field <q>{key}</q> to resource {resource_link} in {pkg_link}"
+msgstr "Feld <q>{key}</q> hinzugefügt zur Ressource {resource_link} in {pkg_link}"
+
+#: ckanext/switzerland/templates/snippets/changes/resource_extras.html:26
+msgid "Added the following fields to resource {resource_link} in {pkg_link}"
+msgstr ""
+"Die folgenden Felder wurden zur Ressource {resource_link} in {pkg_link} "
+"hinzugefügt"
+
+#: ckanext/switzerland/templates/snippets/changes/resource_extras.html:35
+msgid "{key} with value {value}"
+msgstr "{key} mit dem Wert {value}"
+
+#: ckanext/switzerland/templates/snippets/changes/resource_extras.html:40
+#: ckanext/switzerland/templates/snippets/changes/resource_extras.html:67
+msgid "{key}"
+msgstr "{key}"
+
+#: ckanext/switzerland/templates/snippets/changes/resource_extras.html:49
+msgid "Removed field <q>{key}</q> from resource {resource_link} in {pkg_link}"
+msgstr ""
+"Feld <q>{key}</q> wurde von der Ressource {resource_link} in {pkg_link} "
+"entfernt"
+
+#: ckanext/switzerland/templates/snippets/changes/resource_extras.html:59
+msgid "Removed the following fields from resource {resource_link} in {pkg_link}"
+msgstr ""
+"Die folgenden Felder wurden von der Ressource {resource_link} in "
+"{pkg_link} entfernt"
+
+#: ckanext/switzerland/templates/snippets/changes/resource_extras.html:75
+msgid ""
+"Changed value of field <q>{key}</q> of resource {resource_link} to "
+"<q>{new_val}</q> (previously <q>{old_val}</q>) in {pkg_link}"
+msgstr ""
+"Wert des Feldes <q>{key}</q> der Resource {resource_link} wurde geändert "
+"zu <q>{new_val}</q> (hatte den Wert <q>{old_val}</q>) in {pkg_link}"
+
+#: ckanext/switzerland/templates/snippets/changes/resource_extras.html:87
+msgid ""
+"Changed value of field <q>{key}</q> to <q>{new_val}</q> in resource "
+"{resource_link} in {pkg_link}"
+msgstr ""
+"Wert des Feldes <q>{key}</q> wurde geändert zu <q>{new_val}</q> in der "
+"Ressource {resource_link} in {pkg_link}"
+
```

Comment:
this is the string that is translated wrongly in CKAN and causes errors (the msgstr has an extra variable that is not in the msgid). Double checked that this is our correct translation. ;)